pinot-broker/src/main/java/org/apache/pinot/broker/broker/AccessControlFactory.java

```diff
@@ -29,6 +31,7 @@
   public static final String ACCESS_CONTROL_CLASS_CONFIG = "class";
 
   public abstract void init(PinotConfiguration confguration);
+  public abstract void init(ZkHelixPropertyStore<ZNRecord> propertyStore);
```

Comment:
See suggestion below. I'd make this a non-abstract method that takes both `PinotConfiguration` and `ZkHelixPropertyStore` and calls the legacy `init(PinotConfiguration)`. Then, add override this method in your new code while old code remains unaffected. You'll also want to document this intended use in the javadoc

Reply:
I think your suggestion is more rational. I have accepted your suggestion. Please check it again.